examples/basic/variables.tf

```diff
@@ -0,0 +1,10 @@
+variable "do_token" {
```

Comment:
It's fine, but please add `default = null` so provider fallbacks to searching env variables.

Reply:
added